README.rst

```diff
@@ -77,7 +77,7 @@ to set the student ID, which defaults to 1::
     http://127.0.0.1:8000/?student=17
 
 Different students will see different student state, for example, while seeing
-the same content.  Student ids are strings, even if they contain only digits
+the same content.  Student IDs are strings, even if they contain only digits
 as the default does.
```

Comment:
This isn't your wording, but it uses implicit grammar, which I find confusing. Maybe something more like: